src/main/java/services/notification/NotificationAdder.java

```diff
@@ -0,0 +1,23 @@
+package services.notification;
+
+import services.event_creation.CalendarEventModel;
+import services.task_creation.TodoListTaskCreationModel;
+
+public class NotificationAdder {
+
+    /***
+     * Create an event notification using the date from eventData and the event id eventId
+     * @param eventData the event data from the user
```

Comment:
Instead of saying "event data from the user", how about "event data passed down from the user interface when the user creates a new event"?

I think it's clearer like that.